ENIGMAsystem/SHELL/Universal_System/Extensions/VideoPlayer/Makefile

```diff
@@ -0,0 +1,14 @@
+SOURCES += Universal_System/Extensions/VideoPlayer/videoplayer.cpp
+ifeq ($(TARGET-PLATFORM), Windows)
+	override LDLIBS += -Wl,-Bdynamic `pkg-config --libs --cflags mpv` -Wl,-Bstatic
+else ifeq ($(TARGET-PLATFORM), MacOSX)
+	SOURCES += Universal_System/Extensions/VideoPlayer/videoplayer.mm
+	override LDLIBS += -Wl,-Bdynamic `pkg-config --libs --cflags mpv` -Wl,-Bstatic
+	override CXXFLAGS += -ObjC
+	override CFLAGS += -ObjC
+	override LDLIBS += -framework Cocoa
+else ifeq ($(TARGET-PLATFORM), Android)
+	override LDLIBS += -Wl,-Bdynamic `pkg-config --libs --cflags mpv` -Wl,-Bstatic
+else # Linux, FreeBSD
+	override LDLIBS += `pkg-config --libs --cflags mpv`

```

Comment:
Well, I have absolutely no idea why azure is getting undefined references but appveyor is not. They both should be working, looking at this Makefile.

Reply:
its because the linux pkg config is getting called you need to use $(pkg-config) var so that  it calls the correct one.

Reply:
Could you give an example of this that already exists somewhere in the engine that I could use as a reference?

Reply:
I don't remember where. I think you just put $(PKG-CONFIG) in the makefile where you have pkg-config it may or may not need to be all caps

Reply:
I discovered a bug with Mac. The video end playback event doesn't get triggered at the end of video playback until after you press the keyboard or mouse. I'm just going to simulate a mouse button on the window when current position equals duration because I don't care enough to go with a more correct solution. If you or Josh feel differently I act accordingly I just don't want to do it unless I have to. When that's done, I'll check out what you said.

Reply:
Nevermind, I figured out the right way to do it that doesn't involve simulating a mouse press.